market_analysis/__init__.py

```diff
@@ -0,0 +1,19 @@
+from pyramid.config import Configurator
+import os
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+    # import pdb; pdb.set_trace()
+    if 'sqlalchemy.url' not in settings:
+        settings["sqlalchemy.url"] = os.environ["DATABASE_URL"]
```

Comment:
I like this approach that allows a database URL supplied in a settings to serve, but uses the env one if it isn't there.